pnpm-lock.yaml

```diff
@@ -2083,6 +2083,36 @@ packages:
       typescript: 5.2.2
     transitivePeerDependencies:
       - supports-color
+    dev: false
+
+  /@typescript-eslint/eslint-plugin@6.11.0(@typescript-eslint/parser@6.11.0)(eslint@8.53.0)(typescript@5.3.3):
```

Comment:
Oof, this isn't going well. I'd just leave your PR as it was and we can do a big dep update elsewhere.